trustchain-http/src/middleware.rs

```diff
@@ -19,7 +19,10 @@ pub async fn validate_did(
 ) -> Result<impl IntoResponse, impl IntoResponse> {
     tracing::info!(did);
     // Validate length is 59 (testnet) or 54 (mainnet)
```

Comment:
Consider removing this as the validation is incorporated into a resolver's role.

Reply:
I don't see any reason not to do some simple checks in the middleware.

Would be good to remove the magic numbers 54 & 59 though, instead having a const for the length of a DID and then add the length of either `did:ion` or `did:ion:test`.

Reply:
Resolved